metricbeat/mb/mb.go

```diff
@@ -206,24 +205,20 @@ func (b *BaseMetricSet) HostData() HostData {
 // the metricset fetches not only the predefined fields but add alls raw data under
 // the raw namespace to the event.
 type ModuleConfig struct {
-	Hosts      []string                `config:"hosts"`
-	Period     time.Duration           `config:"period"     validate:"positive"`
-	Timeout    time.Duration           `config:"timeout"    validate:"positive"`
-	Module     string                  `config:"module"     validate:"required"`
-	MetricSets []string                `config:"metricsets" validate:"required"`
-	Enabled    bool                    `config:"enabled"`
-	Filters    processors.PluginConfig `config:"filters"`
```

Comment:
So `filters` is now `processors`? I think it would be beneficial to users to `Validate()` that `filters` is not set. If it is set then include in the error that they should be using `processors`.